### PR TITLE
Add Agents/Channels tabs

### PR DESCRIPTION
**Pull request by joshbwlng**
Change-type: minor
Signed-off-by: Josh Bowling <josh@balena.io>

***

- Add a new "Agents" tab to the `Channel` lens that displays all users linked to the channel as an agent.
- Add a new "Channels" tab to the `MyUser` lens that display all channels the user is an agent of.

Depends on:
- https://github.com/product-os/jellyfish-client-sdk/pull/578

### CODE REVIEW

File: apps/ui/lib/lens/full/MyUser/MyUser.tsx [hunk 1]
```diff
@@ -292,6 +313,23 @@ export default class MyUser extends React.Component<Props, State> {
 							/>
 						</Flex>
 					</Tab>,
+					<Tab title="Channels" key="channels">
```
Comment (LucianBuzzo) [inline apps/ui/lib/lens/full/MyUser/MyUser.tsx:316]: I feel like this needs a different title, something that implies that you are an agent for the channel 🤔

Comment (joshbwlng) [reply apps/ui/lib/lens/full/MyUser/MyUser.tsx:316]: `Assigned Channels`?

Comment (LucianBuzzo) [reply apps/ui/lib/lens/full/MyUser/MyUser.tsx:316]: @joshbwlng Yup, that will do for now I think

**Comment by joshbwlng** [comment]
Moved to https://github.com/product-os/jellyfish/pull/8491

**Pull request closed without merging**